src/Entity/Todo.php

```diff
@@ -28,7 +28,7 @@ class Todo
     private \DateTimeInterface $created_at;
 
     /**
-     * @ORM\Column(type="datetime")
+     * @ORM\Column(type="datetime", nullable=true)
      */
     private \DateTimeInterface $completed_at;
```

Comment:
Your property allow null but u always waiting DateTimeInterface so read it https://www.php.net/manual/ru/language.oop5.properties.php then change